assemblyuploader/ena_queries.py

```diff
@@ -74,8 +74,8 @@ def __init__(self, accession, username=None, password=None):
         self.url = ENA_API_URL
         self.accession = accession
         self.acc_type = parse_accession(accession)
-        username = username or os.getenv("ENA_API_USER")
-        password = password or os.getenv("ENA_API_PASSWORD")
+        username = username or os.getenv("WEBIN_USERNAME")
+        password = password or os.getenv("WEBIN_PASSWORD")
```

Comment:
Do we want to use the same as the genome uploader? It's currently ENA_WEBIN and ENA_WEBIN_PASSWORD but I can also change those into these